stream.js

```diff
@@ -59,9 +59,10 @@ Stream.prototype._writeToSink = function (data) {
     this.sink.write(this.cursor)
 }
 
+// returns true="next block", false="end stream", null="pause"
 Stream.prototype._handleBlock = function(block) {
   while (true) {
-    if (this.sink.paused) return false
+    if (this.sink.paused) return null
```

Comment:
I'm wondering why exactly this is different from what it was before? I mean after was a this.sink.paused check instead of the null check, so was the a concurrency bug where the value of this.sink.paused was changed between the return and the else if check?

Reply:
This was so tricky, and it was so late at night, that I can't quite remember the reason. Wanna try changing it to false and seeing what happens?

Reply:
Yes, I'll try that :)

Reply:
Interesting it didn't work before. Wow tricky bug!